### DISCUSSION TITLE
Running the dev server with NODE_ENV=test and the NODE_ENV remains development

### DISCUSSION THREAD

**Discussion by razb-viola** [vercel/next.js#61937] [Help]
### Summary

I want to run my next.js app in test mode - so it will interact with a mock backend and the NODE_ENV will be set to test, instead of development when running `next dev`.

However, when I set this command, it only takes the .env.test file but the NODE_ENV remains development.
I have tried to override the NODE_ENV from the .env.test file as follow: NODE_ENV='test'. However it didn't work.

How can I set this up?

### Additional information

```js
I want my frontend team to develop the frontend web app, yet the backend is not ready. So I point them to the mock server, which is must shorter to implement, rather than a full backend implementation.
```

**Comment by icyJoseph** [comment]
Hi,

It might be worth just using a different variable, `ENVIRONMENT` or something like that. `NODE_ENV` having specific values is often as assumption made by frameworks. 

 - https://overreacted.io/how-does-the-development-mode-work/

**Reply by razb-viola** [reply]
Ok sure, thanks
